### ISSUE TITLE
Missing openpyxl requirement

### ISSUE THREAD

**Issue by alex-gilman** [django-import-export/django-import-export#2004]
The admin module requires openpyxl, but openpyxl is now an optional requirement.

To reproduce:
- install 4.3.0
- import import_export.admin.ImportExportModelAdmin
- observe ModuleNotFoundError for openpyxl

Versions:
 - Django Import Export: 4.3.0
 - Python 3.11.10
 - Django 4.2.16

Expected behavior:
ImportExportModelAdmin should be imported without error

**Comment by matthewhegarty** [comment]
To reproduce:

```bash
python -m venv venv
pip install django-import-export
tests/manage.py runserver
```

Workaround:

```bash
pip install django-import-export[xlsx]
```

**Comment by matthewhegarty** [comment]
Thanks for raising.  Fixed in [4.3.1](https://django-import-export.readthedocs.io/en/stable/changelog.html#id1)

**Comment by alex-gilman** [comment]
Verified that 4.3.1 is working for us. Thank you for addressing so quickly!
